packages/web-api/src/types/request/chat.ts

```diff
@@ -1,17 +1,18 @@
-import type {
-  KnownBlock,
-  Block, // TODO: these will be combined into one in a new types release
-  MessageAttachment,
-  LinkUnfurls,
-  MessageMetadata,
-} from '@slack/types';
+import { OptionalArgument } from '../helpers';
+
 import type {
   CursorPaginationEnabled,
   OptionalTeamAssignable,
   TimelinePaginationEnabled,
   TokenOverridable,
 } from './common';
-import { OptionalArgument } from '../helpers';
+import type {
+  Block, // TODO: these will be combined into one in a new types release
+  KnownBlock,
+  LinkUnfurls,
+  MessageAttachment,
+  MessageMetadata,
+} from '@slack/types';
```

Comment:
This ordering seems unexpected to me - I'd've thought `@slack/types` finds a place before `./*` imports - but no blocker if this is what the linter likes 🙏 

Also only noticed this in a few `web-api` files... Not sure why though!

Reply:
I'm also not sure why.... but I almost don't want to ask questions. Just tell me how to order the damn things and don't make me think!